### PR TITLE
[73] clear content after success and clear button click

### PR DESCRIPTION
**Pull request by oaedward**
# PR Description
Clear field with the Clear Button in the forms

### CODE REVIEW

File: src/client/web/src/components/BIOMD/BIOMD/BIOMD-MANUFACTURER/MANUFACTURER.vue
Comment (akshaypuli) [inline src/client/web/src/components/BIOMD/BIOMD/BIOMD-MANUFACTURER/MANUFACTURER.vue:1]: The following fields did not clear 
image

File: src/client/web/src/components/BIOMD/BIOMD/BIOMD-MODEL/MODEL.vue
Comment (akshaypuli) [inline src/client/web/src/components/BIOMD/BIOMD/BIOMD-MODEL/MODEL.vue:1]: The following field did not clear
image

File: src/client/web/src/components/BIOMD/BIOMD/BIOMD-VENDOR/VENDOR.vue
Comment (akshaypuli) [inline src/client/web/src/components/BIOMD/BIOMD/BIOMD-VENDOR/VENDOR.vue:1]: The following fields did not clear
image

File: src/client/web/src/components/BIOMD/BIOMD/BIOMD-FACILITY/FACILITY.vue
Comment (akshaypuli) [inline src/client/web/src/components/BIOMD/BIOMD/BIOMD-FACILITY/FACILITY.vue:1]: The following fields did not clear
image

**Review by akshaypuli — CHANGES_REQUESTED** [review]
@oaedward  It looks like most of the country, province and district fields did not clear. Maybe it is the same bug in all the components

File: src/client/web/src/components/BIOMD/BIOMD/BIOMD-FACILITY/FACILITY.vue
Comment (oaedward) [reply src/client/web/src/components/BIOMD/BIOMD/BIOMD-FACILITY/FACILITY.vue:1]: I think I know why:
The fields cleared after the Country API was selected as well as all the other APIs. If the content was manually entered into the field, the clear button had no effect. The user had to select from the API options.

I will check on that.

Comment (oaedward) [reply src/client/web/src/components/BIOMD/BIOMD/BIOMD-FACILITY/FACILITY.vue:1]: All the fields which were unable to clear were tied to the API. Once an object is selected from the options provided by the API, the `clear` button works. 

However, the clear button did not work when the fields received a manual random entry. 

I will check on that, however.

Comment (akshaypuli) [reply src/client/web/src/components/BIOMD/BIOMD/BIOMD-FACILITY/FACILITY.vue:1]: Can we change those fields to select box or combo box?

Comment (oaedward) [reply src/client/web/src/components/BIOMD/BIOMD/BIOMD-FACILITY/FACILITY.vue:1]: They are actually combo boxes but if the country list isn't populated into the database, it looks just like a normal input field. 

The district and region fields are also dependent on the country. I think a good implementation will be to disable those fields until a country is selected.

**Comment by akshaypuli** [comment]
Closing as per discussion on 8/27/2023

**Pull request closed without merging**